src/components/layouts/App.js

```diff
@@ -138,103 +138,37 @@ function App() {
 			</Suspense>
 			{fetchingUser ? (
 				<div className="vh-100">
-					<Loader header="signing in" spinner />
+					<Loader header="Signing in..." spinner />
 				</div>
-			) : loggedIn ? (
-				<main id="app-container">
-					<ErrorBoundary FallbackComponent={ErrorFallback}>
-						<Suspense
-							fallback={
-								<div className="auth-centered-container">
-									<Loader header=" " spinner />
-								</div>
-							}
-						>
-							{/* can we put instance routes in here, each in a suspense tag (since they're lazily loaded) */}
+			) : (
+				<main id={loggedIn ? 'app-container' : ''} className={!loggedIn ? 'auth-container' : ''}>
```

Comment:
Not the most ideal and it works buttttt if ya'll have any better suggestions please let me know! 🙂